concent_api/core/tests/utils.py

```diff
@@ -283,7 +283,11 @@ def _get_deserialized_task_to_compute(
                 want_to_compute_task=want_to_compute_task if want_to_compute_task is not None else self._get_deserialized_want_to_compute_task(
                     provider_public_key=provider_public_key,
                     provider_ethereum_public_key=provider_ethereum_public_key,
-                    task_id=task_id,
+                    task_id=(
+                        compute_task_def['task_id']
```

Comment:
this part seems unnecessary here - if `compute_task_def` is not passed as argument, it is created with the same `task_id`

Reply:
It was not working this way, that's why I have changed it.

Reply:
what exactly is not working?

Reply:
You mean it wasn't working when CTD was already created and we create TTC with new task_id?

Reply:
I see, but when I looked at this first time it looked strange for me as well. `compute_task_def` is passed to this method or created, so in this place it exists. Is it possible that `compute_task_def` does not have field `['task_id']`? Could it be changed simple to 
`task_id=compute_task_def['task_id']` ?

BTW. :+1: for discovering it